readme.md

```diff
@@ -80,40 +95,46 @@ Likewise when we write queries, we use **Verbs** to describe the specific data
 we want.
 
 Essentially, in order to store and retrieve information, a lot of what we do
-today will look like some form of the equation:
+today in ruby will look like some form of the equation:
```

Comment:
Be consistent with whether you capitalize Ruby or not. It's used in different ways throughout the lesson.